api-reference/beta/api/directory_post_deleteditems.md

```diff
@@ -0,0 +1,74 @@
+# Restore deletedItem
+
+Restores a [deleted item](../resources/directory.md).
+
+> **Important**: Deleted groups in the Recycle bin will be permanently deleted after 30 days.
```

Comment:
I think this is fine.  Not sure about calling it a "recycle bin", but I'm not too fussed.  Should Recycle be capitalized?